swap-network.js

```diff
@@ -0,0 +1,47 @@
+export async function addLuksoL16Testnet() {
+  try {
+    // Open request to add custom network
+    await window.ethereum.request({
+      method: 'wallet_addEthereumChain',
+      params: [
+        {
+          chainId: '0xB0C',
+          chainName: 'LUKSO L16',
+          nativeCurrency: {
+            name: 'LUKSO',
+            symbol: 'LYXt',
+            decimals: 18,
+          },
+          rpcUrls: ['https://rpc.l16.lukso.network'],
```

Comment:
this: `https://rpc.l16.lukso.network`

should go in a global conf variable.
I'm sure this value is already somewhere in the codebase